lib/data_completeness.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+# The completeness of each type of member data as a percentage
+class DataCompleteness
+  # @param [Array PersonCard]
+  def initialize(people:)
```

Comment:
If this is receiving an array of` `PersonCard`, perhaps better to call this something like `person_cards`. `people` would imply to me that it's wanting `Person` objects instead.